daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ConcurrentCompiledPackages.scala

```diff
@@ -23,16 +23,20 @@ final class ConcurrentCompiledPackages extends MutableCompiledPackages {
   private[this] val _packageDeps: ConcurrentHashMap[PackageId, Set[PackageId]] =
     new ConcurrentHashMap()
   private[this] var _profilingMode: speedy.Compiler.ProfilingMode = speedy.Compiler.NoProfile
+  private[this] var _stackTraceMode: speedy.Compiler.StackTraceMode = speedy.Compiler.FullStackTrace
 
   def getPackage(pId: PackageId): Option[Package] = _packages.get(pId)
   def getDefinition(dref: speedy.SExpr.SDefinitionRef): Option[speedy.SExpr] =
     Option(_defns.get(dref))
 
-  def stackTraceMode = speedy.Compiler.FullStackTrace
-  def profilingMode = _profilingMode
+  override def profilingMode = _profilingMode
```

Comment:
This is unrelated. I've just learnt about from @SamirTalwar-DA in the change below and though it might be a good idea to _immediately_ apply my lesson here as well, given how small the PR is.